getit/getit.py

```diff
@@ -5,7 +5,11 @@
 from getopt import GetoptError, getopt
 from getpass import getuser
 from time import time
-from urllib.request import urlopen
+from contextlib import closing
+try:
+    from urllib.request import urlopen
+except ImportError:
+    from urllib2 import urlopen
 
 from colorama import Fore, Style, init
 init()
```

Comment:
check line 249, You will find `input()` syntax which is available only in python3